src/middlewared/middlewared/plugins/crypto.py

```diff
@@ -1111,6 +1157,13 @@ def __create_internal(self, job, data):
     )
     @job(lock='cert_update')
     async def do_update(self, job, id, data):
```

Comment:
@william-gr do we need trivial docs like this? Maybe we shouldn't say anything if there is nothing to say? E.g. if creating/updating object has no restrictions.or unexpected behavior.

Reply:
@thenylogin We do. What is trivial to you may not be trivial to someone else totally unfamiliar with the API. This is not a good example though, we should say what the method does and point out any particularities of what the attributes mean for easier understand of API user.

@sonicaj I had asked for you to check oit how its done in pool plugin. This is not python docstring, its the API documentation (/api/docs). These keywords you are using are not understood.